__tests__/lib/style-formatting/multipart-form-data.test.js

```diff
@@ -0,0 +1,428 @@
+
+    it.each([
+      [
+        'should NOT support space delimited multipart/form-data styles for non exploded empty input',
+        bodyNoExplode,
+        { body: { primitive: emptyInput } },
+        [],
+      ],
+      [
+        'should NOT support space delimited multipart/form-data styles for exploded empty input',
+        bodyExplode,
+        { body: { primitive: emptyInput } },
+        [],
+      ],
+      [
+        'should NOT support space delimited multipart/form-data styles for non exploded string input',
+        bodyNoExplode,
+        { body: { primitive: stringInput } },
+        [],
+      ],
+      [
+        'should NOT support space delimited multipart/form-data styles for exploded string input',
+        bodyExplode,
+        { body: { primitive: stringInput } },
+        [],
+      ],
+      [
+        'should support space delimited multipart/form-data styles for non exploded array input',
+        bodyNoExplode,
+        { body: { array: arrayInput } },
+        [{ name: 'array', value: 'blue black brown' }],
+      ],
+      [
+        'should support space delimited multipart/form-data styles for non exploded array input and NOT encode already encoded values',
+        bodyNoExplode,
+        { body: { array: arrayInputEncoded } },
+        [{ name: 'array', value: 'something%26nothing%3Dtrue hash%23data' }],
+      ],
+      [
+        'should NOT support space delimited multipart/form-data styles for exploded array input',
+        bodyExplode,
+        { body: { array: arrayInput } },
+        [],
+      ],
+      // This is supposed to be supported, but the style-serializer library we use does not have support. Holding off for now.
+      /* [
+        'should support space delimited multipart/form-data styles for non exploded object input',
+        bodyNoExplode,
+        { body: { object: objectInput } },
+        // Note: this is space here, but %20 in the example above, because encoding happens far down the line
+        { object: 'R 100 G 200 B 150' },
+      ],
+      [
+        'should NOT support space delimited multipart/form-data styles for exploded object input',
+        bodyExplode,
+        { body: { object: objectInput } },
+        [],
+      ], */
+    ])('%s', async (_, operation = {}, formData = {}, expectedRequestBody = undefined) => {
+      const oas = createOas('/body', operation);
+      const har = oasToHar(oas, oas.operation('/body', 'post'), formData);
+      await expect(har).toBeAValidHAR();
+
+      // Note: not sure what the best path forward is here. research har spec more
+      // Basically some tests in this section, when the test is a "should NOT" test, give us an empty array, and some don't give us postData at all
+      // we need to standardize them
+      if (expectedRequestBody === undefined) {
+        // eslint-disable-next-line jest/no-conditional-expect
+        expect(har.log.entries[0].request.postData).toBeUndefined();
```

Comment:
Yeah this is tricky because we don't want to not have `postData` present because that'll lose the `mimeType`.